modules/FvwmMFL/FvwmMFL.c

```diff
@@ -113,6 +114,7 @@ static struct fvwm_msg *fvwm_msg_new(void);
 static void fvwm_msg_free(struct fvwm_msg *);
 static void register_interest(void);
 static void send_version_info(struct client *);
+static char *set_socket_path();
```

Comment:
```suggestion
static char *set_socket_path(void);
```

This at least makes it >C89 compliant.  ;)